### ISSUE TITLE
Create support for polling

### ISSUE THREAD

**Issue by adriatic** [aurelia-community/community-wide-collaboration#5]
This feature is quite relevant to achieve the right start as almost everything that is being done for our **[website](https://github.com/aurelia-community/website)** project depends on a solid prioritization.

**Comment by bigopon** [comment]
@adriatic Can I know where to find the spec of this feature ?

**Comment by adriatic** [comment]
@bigopon  yes, of course - the plan for AUCS was and will always based on open commmunication between peers, where some of these peers will take on some additional roles (publisher, blog admin, project lead) as vaguely described in **[Aurelia Users Community Space specific role](https://github.com/aurelia-community/community-wide-collaboration/issues/6)**, as well as some initial decisions that will also be made by surveys, designed to provide best answers to the questions raised in the following articles

- **[Types of AUCS projects](https://github.com/aurelia-community/community-wide-collaboration/issues/8)**
- **[List of initial AUCS projects](https://github.com/aurelia-community/community-wide-collaboration/issues/7)**
- **[Technology stack proposal ](https://github.com/aurelia-community/community-wide-collaboration/issues/3)**
- **[Choice of document editor](https://github.com/aurelia-community/community-wide-collaboration/issues/9)**

My current task is pretty crazy, as it resembles running multiple threads without any semaphores to ensure the meaningfulness of the conclusions to be derived.

Back to your question on Surveys: 

![image](https://user-images.githubusercontent.com/2712405/35240780-92942232-ff82-11e7-86b4-dcdaa8c33959.png)

Like all other "built in" components surveys are done via a "real component" hosted in an iFrame. This allowed the minimum amount of work on the AUCS Portal, leaving the state management to "embedded" tools

At the moment, we are using https://www.surveyhero.com as the tool for managing surveys
